icssc/AntAlmanac
# Add dark mode versions for Course Pane's "Loading" animation and "No Results" picture

## Summary
Previously the loading animation was a video. Now it's a gif.
Also added a second gif for dark mode.

(Sorry the gifs are kind of quick)
![light](https://user-images.githubusercontent.com/29494270/117528189-c7a62e00-af85-11eb-8785-96f0d2a995ce.gif)
![dark](https://user-images.githubusercontent.com/29494270/117528192-ca088800-af85-11eb-99dd-2db8a8489f91.gif)

Additionally, added a dark mode image for "No Results"
![image](https://user-images.githubusercontent.com/29494270/117528981-ce36a480-af89-11eb-86a3-d5d420a506d3.png)

## Test Plan
Loading Animation
- Search for a class in light mode
- Search for a class in dark mode
- Verify that the loading animation is shown correctly for both

No Result Picture
- Search with no valid queries in light mode
- Search with no valid queries in dark mode
- Verify that the "No results" picture shows correctly for both

## Issues
Relates to #155
